sernet.gs.service/src/sernet/gs/scraper/GSScraper.java

```diff
@@ -366,7 +373,14 @@ private void getStand(String kapitel, Node root) throws XPathException {
 		} catch (XPathException e) {
 			Logger.getLogger(GSScraper.class).error(e);
 			throw new GSServiceException(e);
-		}
+        } catch (GSServiceException e) {
+            if (LOG.isDebugEnabled()) {
```

Comment:
Please log message and stack trace. Moreover use log level error if kapitel is not equals "b00000". Use a constant for string "b00000".